content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -95,15 +98,18 @@ ufw allow from 10.42.0.0/16 to any #pods
 ufw allow from 10.43.0.0/16 to any #services
 ```
 
-You may also need to open other ports if your setup requires them. See the K3s
-[Inbound rules](https://docs.k3s.io/installation/requirements?_highlight=red&_highlight=hat&os=debian#inbound-rules-for-k3s-nodes)
-docs for more information. If you change the default CIDR for pods or services,
-you must update the firewall rules accordingly.
+You should also add [port rules](https://ubuntu.com/server/docs/firewalls)
+for all the [RDI services]({{< relref "/integrate/redis-data-integration/ingest/reference/ports" >}})
+you intend to use:
+
+```bash
+ufw allow 8080/tcp  # (Required) rdi-operator/rdi-api
+ufw allow 9090/tcp  # vm-dis-reloader
+ufw allow 9092/tcp  # prometheus-service
+ufw allow 9121/tcp  # rdi-metric-exporter
+```
 
-See
-[Rancher support and maintenance terms](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-6/)
-for more information about the OS versions that have been tested with
-Rancher-managed K3s clusters.
+You may also need to add similar rules to open other ports if your setup requires them.
```

Comment:
Add `> Note:` or just `>`

Reply:
Done (there's also another instance of this later in the file, btw).